### PR TITLE
Add tutorial for taup-velplot

### PR DESCRIPTION
**Pull request by seisman**
**Description of proposed changes**

Address #1 

**PRs for new software/tutorials**

- [x] Software metadata is complete
- [x] The "last updated date" in the metadata page is up-to-date
- [x] All links work
- [x] You're listed as the author(s) in the metadata page
- [ ] Anyone who have made significant contributions (e.g., review) to
      the tutorials are listed in the meteda page
- [ ] Add a test to weekly CI jobs

**PRs for bugs/improvements**

- [ ] You're listed as the proofreader in the metadata page

### CODE REVIEW

File: source/taup/taup_velplot.rst [hunk 1]
```diff
@@ -0,0 +1,27 @@
+taup velplot
+============
+
+:本节贡献者: |田冬冬|\（作者）、
+             |姚家园|\（审稿）
+:最近更新日期: 2021-06-01
+
+----
+
+``taup velplot`` 可以很方便地生成一个可用于绘制速度-深度剖面的 GMT 脚本。
```
Comment (core-man) [inline source/taup/taup_velplot.rst:10]:
```suggestion
``taup velplot`` 可以很方便地生成用于绘制速度-深度剖面的 GMT 脚本。
```

File: source/taup/taup_velplot.rst [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+使用 ``taup velplot --help`` 可查询该命令的完整用法和选项。此处仅介绍其常用用法。
+
+绘制 IASP91 参考模型的速度-深度剖面::
```
Comment (core-man) [inline source/taup/taup_velplot.rst:14]:
```suggestion
绘制 iasp91 参考模型的速度-深度剖面::
```

File: source/taup/taup_velplot.rst [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+绘制 iasp91 参考模型的速度-深度剖面::
+
+    $ taup velplot -mod iasp91
+
+执行该命令会生成脚本 :file:`iasp91_vel.gmt`\ 。执行该脚本即可生成 PDF 格式的
+速度-深度剖面::
+
+    $ sh iasp91_vel.gmt
+
+绘制出来的图件效果如下。其中蓝线为 P 波速度，红线为 S 波速度。
```
Comment (core-man) [inline source/taup/taup_velplot.rst:23]:
```suggestion
绘制出来的图件效果如下（蓝线为 P 波速度，红线为 S 波速度）：
```

File: source/taup/taup_velplot.rst [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+             |姚家园|\（审稿）
+:最近更新日期: 2021-06-01
+
+----
+
+``taup velplot`` 可以很方便地生成用于绘制速度-深度剖面的 GMT 脚本。
+
+使用 ``taup velplot --help`` 可查询该命令的完整用法和选项。此处仅介绍其常用用法。
+
+绘制 iasp91 参考模型的速度-深度剖面::
```
Comment (core-man) [inline source/taup/taup_velplot.rst:14]: The command is not uses to plot vel-depth but output a script to plot it.

File: source/taup/taup_velplot.rst [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+    $ taup velplot -mod iasp91
+
+执行该命令会生成脚本 :file:`iasp91_vel.gmt`\ 。执行该脚本即可生成 PDF 格式的
```
Comment (core-man) [inline source/taup/taup_velplot.rst:18]: The first setence should be moved to last command above.

File: source/taup/taup_velplot.rst (hunk 4 above)
Comment (seisman) [inline source/taup/taup_velplot.rst:14]:
```suggestion
生成 iasp91 参考模型的速度-深度剖面绘图脚本::
```

Comment (seisman) [reply source/taup/taup_velplot.rst:14]: Please see my suggestion below,.

File: source/taup/taup_velplot.rst [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+``taup velplot`` 可以很方便地生成用于绘制速度-深度剖面的 GMT 脚本。
+
+使用 ``taup velplot --help`` 可查询该命令的完整用法和选项。此处仅介绍其常用用法。
+
+生成 iasp91 参考模型的速度-深度剖面绘图脚本::
+
+    $ taup velplot -mod iasp91
+
+执行该命令会生成脚本 :file:`iasp91_vel.gmt`\ 。执行该脚本即可生成 PDF 格式的
```
[the remark this answers is not in the document]
Comment (core-man) [reply source/taup/taup_velplot.rst:18]: I cannot comment on two lines, please move last line into the line to be one line only.

**Review by core-man — APPROVED** [review]
Revise the updating date if necessary please

File: source/taup/taup_velplot.rst [hunk 7]
```diff
@@ -0,0 +1,27 @@
+taup velplot
+============
+
+:本节贡献者: |田冬冬|\（作者）、
+             |姚家园|\（审稿）
+:最近更新日期: 2021-06-01
```
Comment (seisman) [inline source/taup/taup_velplot.rst:6]:
```suggestion
:最近更新日期: 2021-06-02
```

File: source/taup/taup_velplot.rst [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+``taup velplot`` 可以很方便地生成用于绘制速度-深度剖面的 GMT 脚本。
+
+使用 ``taup velplot --help`` 可查询该命令的完整用法和选项。此处仅介绍其常用用法。
+
+生成 iasp91 参考模型的速度-深度剖面绘图脚本::
+
+    $ taup velplot -mod iasp91
+
+执行以上命令生成的脚本 :file:`iasp91_vel.gmt`\ ，生成 PDF 格式的
+速度-深度剖面::
```
Comment (seisman) [inline source/taup/taup_velplot.rst:19]:
```suggestion
执行以上命令生成的脚本 :file:`iasp91_vel.gmt`\ ，生成 PDF 格式的速度-深度剖面::
```

**Pull request merged**